platform/src/main/java/org/hillview/table/rows/GuessSchema.java

```diff
@@ -33,6 +35,19 @@
  * Helper class to guess the schema of data given a set of strings.
  */
 public class GuessSchema {
+
+    public String name;
```

Comment:
If you add these fields you should remove the API below which still receives a SchemaInfo as an argument and refactor the code that was using it.